back/engines/free/email_campaigns/app/models/email_campaigns/campaigns/community_monitor_report.rb

```diff
@@ -0,0 +1,150 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: email_campaigns_campaigns
+#
+#  id               :uuid             not null, primary key
+#  type             :string           not null
+#  author_id        :uuid
+#  enabled          :boolean
+#  sender           :string
+#  reply_to         :string
+#  schedule         :jsonb
+#  subject_multiloc :jsonb
+#  body_multiloc    :jsonb
+#  created_at       :datetime         not null
+#  updated_at       :datetime         not null
+#  deliveries_count :integer          default(0), not null
+#  context_id       :uuid
+#
+# Indexes
+#
+#  index_email_campaigns_campaigns_on_author_id   (author_id)
+#  index_email_campaigns_campaigns_on_context_id  (context_id)
+#  index_email_campaigns_campaigns_on_type        (type)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (author_id => users.id)
+#
+module EmailCampaigns
+  class Campaigns::CommunityMonitorReport < Campaign
+    include Disableable
+    include Consentable
+    include Schedulable
+    include RecipientConfigurable
+    include Trackable
+    include LifecycleStageRestrictable
+
+    allow_lifecycle_stages only: ['active']
+
+    recipient_filter :user_filter_admins_moderators_only
+    recipient_filter :user_filter_no_invitees
```

Comment:
`    recipient_filter :user_filter_admins_moderators_only
`
Does this also filter for just Community Monitor admins/moderators then?